llm_studio/src/utils/modeling_utils.py

```diff
@@ -496,6 +496,10 @@ def update_backbone_config(config: Any, cfg: Any):
     if "mpt-" in cfg.llm_backbone:
         config.init_device = cfg.environment._device
 
+    if hasattr(config, "pretraining_tp"):
+        logger.info("Setting pretraining_tp of model config to 1.")
```

Comment:
Add link to the discussion in the warning.

Reply:
Dont think thats needed, it is a pretty safe switch
llama2-7b has it even as default

Reply:
ok added comment and switch for lora